src/apps/AppCenter/AppCenter.js

```diff
@@ -74,6 +85,7 @@ class AppCenter extends React.Component {
     }
   }
   updateLocation({ activeTab, openedRepoName }) {
+    console.log('AT', activeTab, 'ORN', openedRepoName)
```

Comment:
`console.log` here. If you wish to leave it here we've a log function that works only in dev environments